teuthology/config.py

```diff
@@ -138,6 +138,8 @@ class TeuthologyConfig(YamlConfig):
         'src_base_path': os.path.expanduser('~/src'),
         'verify_host_keys': True,
         'watchdog_interval': 120,
+        'brewhub_url': 'http://brewhub.devel.redhat.com/brewhub',
+        'brewroot_url': 'http://download.devel.redhat.com/brewroot/packages',
```

Comment:
discussed today in standup: we'll refer to Koji instead of brew, so "kojihub_url", "kojiroot_url".

The Koji packages URL is http://kojipkgs.fedoraproject.org/packages